conda_smithy/configure_feedstock.py

```diff
@@ -480,9 +482,7 @@ def migrate_combined_spec(combined_spec, forge_dir, config):
 
     """
     combined_spec = combined_spec.copy()
-    migrations_root = os.path.join(
-        forge_dir, ".ci_support", "migrations", "*.yaml"
-    )
+    migrations_root = os.path.join(forge_dir, "migrations", "*.yaml")
```

Comment:
```suggestion
    migrations_root = os.path.join(forge_dir, ".ci_support", "migrations", "*.yaml")
```
This needs to stay I think

Reply:
I'll defer to you and @isuruf. I don't know enough about what is happening here.

Reply:
I'm just putting the two PRs side by side to see which changes were made. The other PR was not all ahead of master so some changes are actually reversions I think.

Reply:
Yeah I'm just saying I don't feel confident making these changes as I'm unclear what is happening. If you do, please feel free to just push them to this PR.